charts/tempo-distributed/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 1.8.2
+version: 1.8.3
```

Comment:
Looks like the latest tempo-distributed chart version is `1.8.4` (https://github.com/grafana/helm-charts/blob/main/charts/tempo-distributed/Chart.yaml), so I guess this one will become `1.8.5`?